rust/private/rustc.bzl

```diff
@@ -438,9 +439,17 @@ def construct_arguments(
 
     # Tell Rustc where to find the standard library
     args.add_all(rust_lib_paths, before_each = "-L", format_each = "%s")
-
     args.add_all(rust_flags)
-    args.add_all(getattr(attr, "rustc_flags", []))
+
+    # Data file paths for location expansion
```

Comment:
Nit: Unless you feel strongly about it I'd remove the comment, IMHO it doesn't add much.